src/plugins/gui/amountQuotePlugin.ts

```diff
@@ -242,11 +242,17 @@ export const amountQuoteFiatPlugin: FiatPluginFactory = async (params: FiatPlugi
         const ratePair = `${isoFiatCurrencyCode}_iso:USD`
         const rate = await getHistoricalRate(ratePair, isoNow)
         initialValue1 = div(startingFiatAmount, String(rate), DECIMAL_PRECISION)
-        // Round out a decimals
+
+        if (isLightAccount) {
+          // Round down
+          initialValue1 = floor(initialValue1, 0)
```

Comment:
What's the point of this? The rounding should be the same for light account or not.

Reply:
See the description - in the original implementation it was possible to potentially round up, so I wanted to retain that for full accounts

Reply:
@Jon-edge so why not for light accounts?